solara/components/file_input.py

```diff
@@ -0,0 +1,96 @@
+"""
+FileInput component.
+"""
+
+
+import threading
+from typing import Callable, List, Optional, cast
+
+from ipyvuetify.extra import FileInput as ExtraFileInput
+
+import solara
+import solara.hooks as hooks
+from solara.components.file_drop import FileInfo
+
+
+@solara.component
+def FileInput(
+    label: str = "",
+    on_total_progress: Optional[Callable[[float], None]] = None,
+    on_file: Optional[Callable[[List[FileInfo]], None]] = None,
+    accept: str = "",
+    multiple: bool = True,
```

Comment:
```suggestion
    multiple: bool = False,
```
I think this is a better default, will make the default behaviour similar to FileDrop